src/test/scala/temple/builder/project/ProjectBuilderTestData.scala

```diff
@@ -554,4 +565,15 @@ object ProjectBuilderTestData {
       |  isDefault: true
       |  editable: true
       |""".stripMargin
+
+  val complexTemplefilePrometheusConfig: String =
+    """global:
+    |  scrape_interval: 15s
+    |  evaluation_interval: 15s
+    |scrape_configs:
+    |- job_name: complexuser
+    |  static_configs:
+    |  - targets:
+    |    - complexuser:1025
+    |""".stripMargin
```

Comment:
It’s properly aligned on the addition at [l.309](https://github.com/TempleEight/temple/pull/180/files#diff-2db252505436751f5fb19b1bad244c85R309), but not here?

Reply:
Wish the formatter would sort this 😭

Reply:
Will this be fixed by #181 ?